### PR TITLE
Add CI for Silabs EFR32

### LINKED ISSUE #1971: EFR32 Add CI for lock-app and platform
#### Problem
There is no continuous integration testing for the EFR32 platform

 #### Proposed Solution
Add continuous integration testing for the gn build and the make build of the efr32

### PR DESCRIPTION
**Pull request by jepenven-silabs**
#### Problem
The EFR32 platform didn't have any CI

 #### Summary of Changes
Add CI for the lock-app example on the EFR32MG12 platform

fixes #1971

### CODE REVIEW

File: .github/workflows/examples.yaml [hunk 1]
```diff
@@ -15,14 +15,13 @@
 name: Examples
 
 on:
-    push:
-    pull_request:
+  push:
+  pull_request:
```
Comment (BroderickCarlin) [inline .github/workflows/examples.yaml:19]: It looks like the indentation used within the changes in this file are inconsistent with the rest of the indentation of the file

Comment (woody-apple) [reply .github/workflows/examples.yaml:19]: Agree with this, please let's stick to the same indentation (note: if you use VSCode dev container, it should be consistent)

**Review by woody-apple — CHANGES_REQUESTED** [review]
Please fix indentation, but this is awesome, thank you!

File: .github/workflows/examples.yaml (hunk 1 above)
Comment (jepenven-silabs) [reply .github/workflows/examples.yaml]: Indentation should be fixed

**Review by BroderickCarlin — APPROVED** [review]

**Review by mspang — APPROVED** [review]

**Comment by jepenven-silabs** [comment]
@woody-apple

**Review by woody-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Comment by woody-apple** [comment]
Bypassing restyle, and merging.

Filed this issue: https://github.com/restyled-io/restyler/issues/113

**Pull request merged**